masterlist.yaml

```
@@ -1789,6 +1789,17 @@ plugins:
         util: '[FNVEdit v4.0.3g](https://www.nexusmods.com/newvegas/mods/34703)'
         itm: 27
 
+  - name: 'ZionAntCaveRestored-HonestHearts.esp'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/74706' ]
```

Trailing slash for consistency:

```suggestion
    url: [ 'https://www.nexusmods.com/newvegas/mods/74706/' ]
```